dotcom-rendering/src/components/InteractiveBlockComponent.importable.tsx

```diff
@@ -326,46 +326,76 @@ export const InteractiveBlockComponent = ({
 	const placeholderLinkRef = useRef<HTMLAnchorElement>(null);
 	const [loaded, setLoaded] = useState(false);
 	const { darkModeAvailable } = useConfig();
+
+	// Define some one-time flags
```

Comment:
Is this comment necessary? The flags are well-named and speak for themselves 😀 

```suggestion
	// Define some one-time flags
```